eagleproject/core/blockchain/harvest/transaction_history.py

```diff
@@ -27,22 +27,23 @@
 )
 from core.blockchain.rpc import (
     balanceOf,
-    totalSupply
+    totalSupply,
+    dripper_available
 )
 from core.blockchain.apy import (
     get_trailing_apy,
+    to_apy
 )
-from datetime import (
-    datetime,
-)
+import datetime
```

Comment:
nit: Could probably do `from datetime import ( datetime, timedelta )` instead of importing everything from datatime and then using `datetime.datetime`

Reply:
added